tests/components/starlink/patchers.py

```diff
@@ -12,7 +15,11 @@
 COORDINATOR_SUCCESS_PATCHER = patch.object(
     StarlinkUpdateCoordinator,
```

Comment:
We should patch the library and not the coordinator. The coordinator is a detail of the integration.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations